### ISSUE TITLE
[🐛]Cannot read property `map` of undefined with a Docker image without vulns - CLI 1.382.1

### ISSUE THREAD

**Issue by glours** [snyk/snyk#1363]
- `node -v`: `v13.11.0`
- `npm -v`: `6.13.7`
- `snyk -v`: `1.382.1`
- OS: (e.g. OSX, Linux, Windows, ...): `OSX` and `Docker Debian` image
- Command run: (e.g. `snyk test --all-projects`, `snyk protect`, ...)
`snyk test --docker dockerscanci/scratch:1.0`

### Expected behaviour
```sh
$ snyk test --docker  dockerscanci/scratch:1.0

Testing dockerscanci/scratch:1.0...

Organization:      docker-desktop-test
Package manager:   linux
Project name:      docker-image|dockerscanci/scratch
Docker image:      dockerscanci/scratch:1.0
Licenses:          enabled

✓ Tested 0 dependencies for known issues, no vulnerable paths found.

Note that we do not currently have vulnerability data for your image.
```

### Actual behaviour
```sh
$ snyk test --docker dockerscanci/scratch:1.0
Cannot read property 'map' of undefined
```

### Steps to reproduce

- Install the `1.382.1` version of Snyk CLI
- Run the snyk test command with a Docker image without any vulnerabilities

FYI the issue was not present in version `1.334.0` of the CLI
### Debug log

```
snyk test --docker dockerscanci/scratch:1.0 --debug
  snyk test <ref *1> {
  _: [ 'dockerscanci/scratch:1.0', [Circular *1] ],
  docker: true,
  debug: true,
  callGraphBuilderTimeout: '300'
} +0ms
  snyk:run-test Error running test {
  error: TypeError: Cannot read property 'map' of undefined
      at Object.analyze [as analyzeDynamically] (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/node_modules/snyk-docker-plugin/lib/analyzer/docker-analyzer.ts:89:37)
      at processTicksAndRejections (internal/process/task_queues.js:97:5)
      at getDependencies (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/node_modules/snyk-docker-plugin/lib/index.ts:107:20)
      at async Promise.all (index 1)
      at analyzeDynamically (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/node_modules/snyk-docker-plugin/lib/index.ts:71:50)
      at Object.inspect (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/node_modules/snyk-docker-plugin/lib/index.ts:59:10)
      at Object.inspect (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/src/lib/module-info/index.ts:22:20)
      at Object.getSinglePluginResult (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/src/lib/plugins/get-single-plugin-result.ts:13:47)
      at Object.getDepsFromPlugin (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/src/lib/plugins/get-deps-from-plugin.ts:86:16)
      at assembleLocalPayloads (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/src/lib/snyk-test/run-test.ts:359:18)
} +0ms
  snyk-test Failed to test 1 projects, errors: +0ms
  snyk-test error: FailedToRunTestError: Cannot read property 'map' of undefined
    at runTest (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/src/lib/snyk-test/run-test.ts:171:11)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) +1ms
Error:
Testing dockerscanci/scratch:1.0...

Cannot read property 'map' of undefined
    at test (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/src/cli/commands/test/index.ts:272:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
  snyk analytics {
  args: [
    'dockerscanci/scratch:1.0',
    {
      docker: true,
      debug: true,
      callGraphBuilderTimeout: '300',
      org: undefined,
      showVulnPaths: 'some'
    }
  ],
  command: 'bad-command',
  org: undefined,
  metadata: {
    local: true,
    'error-message': '\n' +
      'Testing dockerscanci/scratch:1.0...\n' +
      '\n' +
      "Cannot read property 'map' of undefined",
    error: 'Error: \n' +
      'Testing dockerscanci/scratch:1.0...\n' +
      '\n' +
      "Cannot read property 'map' of undefined\n" +
      '    at test (/.../.nvm/versions/node/v13.11.0/lib/node_modules/snyk/src/cli/commands/test/index.ts:272:19)\n' +
      '    at processTicksAndRejections (internal/process/task_queues.js:97:5)',
    'error-code': 500,
    command: 'test'
  },
  version: '1.382.1',
  os: 'macOS Catalina',
  nodeVersion: 'v13.11.0',
  standalone: false,
  integrationName: '',
  integrationVersion: '',
  id: 'XxxxxxxX',
  ci: false,
  durationMs: 6277
} +0ms

  snyk Exit code: 2 +0ms
  snyk sending request to: https://snyk.io/api/v1/analytics/cli +0ms
  snyk request body size: 817 +0ms
  snyk gzipped request body size: 480 +0ms
  snyk not using proxy +1ms
```

**Comment by agatakrajewska** [comment]
Hi @glours !! The fix is on the way :-)

**Comment by agatakrajewska** [comment]
@glours the fix is now released in v1.383.1, please let us know any other issues! :-)

**Comment by glours** [comment]
@agatakrajewska my e2e tests are happy now 🎉 
Thanks 👍
